.github/workflows/compliance.yml

```diff
@@ -60,6 +60,14 @@ jobs:
         west config manifest.group-filter -- +ci,-optional
         west update -o=--depth=1 -n 2>&1 1> west.update.log || west update -o=--depth=1 -n 2>&1 1> west.update2.log
 
+    - name: Setup Node.js
+      uses: actions/setup-node@cdca7365b2dadb8aad0a33bc7601856ffabcc48e # v4.3.0
+      with:
+        node-version: "20"
+
+    - name: Install dependencies
+      run: npm i -g --ignore-script dts-linter@0.1.0
```

Comment:
This is still an issue, if your repository is [compromised](https://www.paloaltonetworks.com/blog/cloud-security/npm-supply-chain-attack/). We should really have a locked `package-lock.json` file, and only use `npm ci`.